Formula/gromacs.rb

```diff
@@ -1,27 +1,30 @@
 class Gromacs < Formula
   desc "Versatile package for molecular dynamics calculations"
   homepage "http://www.gromacs.org/"
-  url "https://ftp.gromacs.org/pub/gromacs/gromacs-2019.1.tar.gz"
-  sha256 "b2c37ed2fcd0e64c4efcabdc8ee581143986527192e6e647a197c76d9c4583ec"
-
-  bottle do
```

Comment:
Please leave the bottle block untouched.

Reply:
I put the bottle block again.